<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>Widget</class>
 <widget class="QWidget" name="Widget">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>1199</width>
    <height>627</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Widget</string>
  </property>
  <layout class="QGridLayout" name="gridLayout_3">
   <item row="1" column="2">
    <widget class="QLineEdit" name="leStorageDir">
     <property name="text">
      <string/>
     </property>
    </widget>
   </item>
   <item row="4" column="5">
    <widget class="QPushButton" name="pbSaveAs">
     <property name="text">
      <string>Сохранить как</string>
     </property>
    </widget>
   </item>
   <item row="0" column="0" colspan="8">
    <layout class="QGridLayout" name="gridLayout">
     <item row="1" column="0">
      <widget class="QPushButton" name="pbAddInputFiles">
       <property name="text">
        <string>+</string>
       </property>
      </widget>
     </item>
     <item row="1" column="1">
      <widget class="QPushButton" name="pbRemoveSelected">
       <property name="text">
        <string>-</string>
       </property>
      </widget>
     </item>
     <item row="1" column="2">
      <widget class="QPushButton" name="pbProcess">
       <property name="text">
        <string>Обработать выделенные</string>
       </property>
      </widget>
     </item>
     <item row="1" column="3">
      <spacer name="horizontalSpacer">
       <property name="orientation">
        <enum>Qt::Horizontal</enum>
       </property>
       <property name="sizeHint" stdset="0">
        <size>
         <width>40</width>
         <height>20</height>
        </size>
       </property>
      </spacer>
     </item>
     <item row="3" column="0" colspan="4">
      <widget class="QTableView" name="tvMainTable"/>
     </item>
    </layout>
   </item>
   <item row="3" column="0" colspan="8">
    <layout class="QGridLayout" name="gridLayout_2">
     <item row="0" column="0">
      <widget class="QLabel" name="label">
       <property name="text">
        <string>Лог</string>
       </property>
      </widget>
     </item>
     <item row="1" column="0">
      <widget class="QPlainTextEdit" name="pteLogs"/>
     </item>
     <item row="1" column="1">
      <widget class="QPlainTextEdit" name="pteOutputPreview"/>
     </item>
     <item row="0" column="1">
      <widget class="QLabel" name="label_2">
       <property name="text">
        <string>Предпросмотр итогового файла</string>
       </property>
      </widget>
     </item>
    </layout>
   </item>
   <item row="4" column="3">
    <widget class="QLabel" name="label_4">
     <property name="text">
      <string>Изменения в итоговом файле</string>
     </property>
    </widget>
   </item>
   <item row="1" column="0">
    <widget class="QLabel" name="label_3">
     <property name="text">
      <string> Директория для итоговых файлов</string>
     </property>
    </widget>
   </item>
   <item row="4" column="7">
    <widget class="QPushButton" name="pbCancelEditing">
     <property name="text">
      <string>Отмена</string>
     </property>
    </widget>
   </item>
   <item row="1" column="7">
    <widget class="QPushButton" name="pbChooseDirectory">
     <property name="text">
      <string>...</string>
     </property>
    </widget>
   </item>
   <item row="4" column="4">
    <widget class="QPushButton" name="pbSaveEditing">
     <property name="text">
      <string>Сохранить</string>
     </property>
    </widget>
   </item>
  </layout>
 </widget>
 <resources/>
 <connections/>
</ui>
